resources/known-amendments.md

```diff
@@ -725,6 +610,20 @@ Without this amendment, it is possible to have a [payment path](../docs/concepts
 With this amendment, those payments fail with the [`temBAD_PATH_LOOP` result code](../docs/references/protocol/transactions/transaction-results/tem-codes.md) instead.
 
 
+### fixAMMOverflowOffer
```

Comment:
Should fix AM_M_OverflowOffer come after fixAm_e_ndmentMajorityCalc?

Reply:
No, uppercase letters come before lowercase.